### DISCUSSION TITLE
Transfert Folder

### DISCUSSION THREAD

**Discussion by gringojack** [mytourbook/mytourbook#1049] [General]
Hi
What is "Transfert Folder" for?
Thanks

**Comment by FJBDev** [comment]
Where do you see a "Transfert folder" ?

**Reply by wolfgang-ch** [reply]
It's propably the "Transfer path" which is used when data are imported from a serial port

![image](https://user-images.githubusercontent.com/1283445/231945546-a447e4e7-3912-4287-a2cc-87ce96d6bc69.png)
